docs/streams.md

```diff
@@ -5,91 +5,83 @@ EventStoreDB is purpose-built for event storage. Unlike traditional state-based
+| **`$maxAge`**       | Sets a sliding window based on dates. When data reaches a certain age, it automatically disappears from the stream and becomes eligible for scavenging. This value is set as an integer representing the number of seconds (must be >= 1).                                                                                                                     |
+| **`$maxCount`**     | Sets a sliding window based on the number of items in the stream. When data reaches a certain length, it automatically disappears from the stream and becomes eligible for scavenging. This value is set as an integer representing the count of items (must be >= 1).                                                                                         |
+| **`$cacheControl`** | Controls the cache of the head of a stream. While most URIs in a stream are infinitely cacheable, the head, by default, will not cache. In some situations, it may be preferable to set a small amount of caching on the head to allow intermediaries to handle polls (e.g., 10 seconds). The argument is an integer representing the seconds to cache (must be >= 1). |
+
+&nbsp;
 
 ::: tip 
-If you set both `$maxAge` and `$maxCount` then events will become eligible for scavenging when either
-criteria is met. For example, if you set `$maxAge` to 10 and `$maxCount` to 50,000, events will be marked as
-eligible for scavenging after either 10 seconds, or 50,000 events, have passed. Deleted items will only be
-removed once the scavenge process runs.
+If you configure both **`$maxAge`** and **`$maxCount`**, events then become eligible for scavenging when either
+condition is met. For example, if you set **`$maxAge`** to 10 seconds and **`$maxCount`** to 50,000, events 
+qualify for scavenging after either 10 seconds or the accumulation of 50,000 events. Deleted items are removed once the scavenge process runs.
 :::
 
-Security access control lists are also included in the `$acl` section of the stream metadata.
+&nbsp;
+
+**Security access control lists (ACLs)** are part of the **`$acl`** section in the stream metadata.
+&nbsp;
 
 | Property name | Description                                                 |
 |:--------------|:------------------------------------------------------------|
-| `$r`          | The list of users with read permissions                     |
-| `$w`          | The list of users with write permissions                    |
-| `$d`          | The list of users with delete permissions                   |
-| `$mw`         | The list of users with write permissions to stream metadata |
-| `$mr`         | The list of users with read permissions to stream metadata  |
+| **`$r`**          | Users with read permissions                     |
```

Comment:
Should we be consistent with periods? The next table uses periods in the descriptions. I'm happy to go with no periods because these aren't complete sentences.

Reply:
I'm also happy to go with no periods. I'll update the style guide